src/nile/common.py

```diff
@@ -38,7 +38,13 @@ def get_gateway():
 
     except FileNotFoundError:
         with open(NODE_FILENAME, "w") as f:
-            f.write('{"localhost": "http://127.0.0.1:5050/"}')
+            f.write(
+                """{
+                "localhost": "http://127.0.0.1:5050/",
+                "goerli2": "https://alpha4-2.starknet.io",
+                "integration": "https://external.integration.starknet.io"
+            }"""
+            )
```

Comment:
```suggestion
            networks = {
                "localhost": "http://127.0.0.1:5050/",
                "goerli2": "https://alpha4-2.starknet.io",
                "integration": "https://external.integration.starknet.io"
            }
            f.write(json.dumps(networks, indent=2))
```

I think this makes the code look cleaner.